R/centralization.R

```diff
@@ -481,7 +481,28 @@ centr_clo_tmax <- centralization_closeness_tmax_impl
 #' centr_eigen(g0)$centralization
 #' centr_eigen(g1)$centralization
 #' @cdocs igraph_centralization_eigenvector_centrality
-centr_eigen <- centralization_eigenvector_centrality_impl
+centr_eigen <- function(graph,
+                        directed = FALSE,
+                        scale = deprecated(),
+                        options = arpack_defaults(),
+                        normalized = TRUE) {
+
+  if (lifecycle::is_present(scale)) {
+    lifecycle::deprecate_soft(
+      "2.1.3",
+      "centr_eigen(scale = )",
+      details = "The function always behaves as if `scale` were TRUE.
+      The argument will be removed in the future."
+    )
+  }
+
+  centralization_eigenvector_centrality_impl(
+    graph = graph,
+    directed = directed,
+    options = options,
+    normalized = normalized
+  )
```

Comment:
I would be explicit and pass `scale=TRUE` explicitly here instead of leaving it to the interface generator to provide this default.

Passing this is necessary to prevent a deprecation warning coming form the C core.